Tweak/Core/KayokoCore.h

```diff
@@ -10,19 +10,20 @@
 #import "Views/KayokoView.h"
 #import <Cephei/HBPreferences.h>
 
-static CGFloat const kHeight = 480;
+OBJC_EXTERN KayokoView* kayokoView;
 
-KayokoView* kayokoView;
+OBJC_EXTERN HBPreferences* preferences;
+OBJC_EXTERN BOOL pfEnabled;
 
-HBPreferences* preferences;
-BOOL pfEnabled;
+OBJC_EXTERN NSUInteger pfMaximumHistoryAmount;
+OBJC_EXTERN BOOL pfSaveText;
+OBJC_EXTERN BOOL pfSaveImages;
+OBJC_EXTERN BOOL pfAutomaticallyPaste;
+OBJC_EXTERN BOOL pfAddTranslateOption;
+OBJC_EXTERN BOOL pfAddSongDotLinkOption;
+OBJC_EXTERN BOOL pfPlaySoundEffects;
 
-NSUInteger pfMaximumHistoryAmount;
-BOOL pfSaveText;
-BOOL pfSaveImages;
-BOOL pfAutomaticallyPaste;
-BOOL pfAddTranslateOption;
-BOOL pfAddSongDotLinkOption;
+OBJC_EXTERN CGFloat pfHeightInPoints;
```

Comment:
## C/C++
- It's a common practice to declare variables in headers (.h, .hpp), and implement them in sources (.c, .cpp, .cc, .m, .mm, .x, .xm).

Reply:
I don't always follow common practices. Having variables declared and initialized in two files makes the code cluttered. Please change it back as it was.

Reply:
It's your repo. You can choose which parts to merge by yourself.